framework/audio/engine/iaudioengine.h

```diff
@@ -40,6 +41,15 @@ class IAudioEngine : MODULE_EXPORT_INTERFACE
     using handle = unsigned int;
     using time = float;
 
+    enum class Status {
+        Undefined = 0,
+        Created,
+        Playing,
+        Seeked,
```

Comment:
I'm not sure about the meaning of the "Seeked" status, because I expect that if I did "seek" for a part of a track, then at that moment I am either in a state of pause, or stop, or play.

Reply:
It is not assigned, only a notification

Reply:
Seeked - removed